### PR TITLE
fix batch bind

### PR DESCRIPTION
**Pull request by LinYunMo**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by novain1102 — APPROVED** [review]

File: cocos/core/renderer/core/render-scene.ts [hunk 1]
```diff
@@ -409,6 +409,8 @@ export class RenderScene {
      * @en Add a draw batch of 2d objects, all draw batches attached to the render scene will be submitted for rendering.
      * @zh 增加一个 2D 渲染批次，渲染场景上挂载的所有 2D 渲染批次都会被提交渲染。
      * @param batch The draw batch.
+     * @internal
+     * @deprecated
```
Comment (novain1102) [inline cocos/core/renderer/core/render-scene.ts]: need depreaced?

**Review by SantyWang — APPROVED** [review]

**Review by novain1102 — APPROVED** [review]

File: cocos/core/root.jsb.ts [hunk 2]
```diff
@@ -209,3 +209,15 @@ rootProto.setRenderPipeline = function (pipeline) {
     this._createBatcher2D();
     return ppl;
 }
+
+rootProto.addBatch = function (batch) {
+    console.error('The Draw Batch class is implemented differently in the native platform and does not support calling this interface using jsb.')
```
Comment (SantyWang) [inline cocos/core/root.jsb.ts]:
```suggestion
    console.error('The Draw Batch class is implemented differently in the native platform and does not support this interface.');
```

File: cocos/core/root.jsb.ts [hunk 3]
```diff
@@ -209,3 +209,15 @@ rootProto.setRenderPipeline = function (pipeline) {
...
+}
+
+rootProto.removeBatch = function (batch) {
+    console.error('The Draw Batch class is implemented differently in the native platform and does not support calling this interface using jsb.')
```
Comment (SantyWang) [inline cocos/core/root.jsb.ts]:
```suggestion
    console.error('The Draw Batch class is implemented differently in the native platform and does not support this interface.');
```

File: cocos/core/root.jsb.ts [hunk 4]
```diff
@@ -209,3 +209,15 @@ rootProto.setRenderPipeline = function (pipeline) {
...
+}
+
+rootProto.removeBatches = function () {
+    console.error('The Draw Batch class is implemented differently in the native platform and does not support calling this interface using jsb.')
```
Comment (SantyWang) [inline cocos/core/root.jsb.ts]:
```suggestion
    console.error('The Draw Batch class is implemented differently in the native platform and does not support this interface.');
```

**Review by SantyWang — APPROVED** [review]

**Pull request merged**